lib/dialogue-agent/entity-linking/entity-finder.js

```diff
@@ -96,4 +97,8 @@ function getBestEntityMatch(searchTerm, entityType, candidates) {
     return best;
 }
 
+function _removeParenthesis(str){
+    return str.replace(/ \([\s\S]*?\)/g, '');
```

Comment:
\s is whitespace, and \S is everything but whitespace, so this will match literally everything? Should write `/ \(.*?\)/` instead? To avoid confusing someone who doesn't know RegExp syntax by heart (I had to check MDN...)